test/WebSites/InProcessForwardsCompatWebSite/InProcessWebSite.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Rename this project file.

Reply:
I did this on purpose. We have ~3 or 4 scenarios that depend on the InProcessWebSite name for testing (runtimeconfig.json, exe checks, etc)

Reply:
Hmmmmm, fine.